README.md

```diff
@@ -0,0 +1,44 @@
+![Aragon Protocol](./docs/aragon-protocol.png)
+
+<img align="right" src="https://github.com/aragon/protocol/workflows/CI/badge.svg">
+  <a href="https://github.com/aragon/protocol/actions"/>
+</img>
+
+## Project
+
+#### 👩‍️ [Become an Aragon Protocol guardian](https://ant.aragon.org)
+Aragon Protocol is now live on Ethereum mainnet. You can become a guardian by staking ANT.
+
+#### ⚖ [Check out the Aragon Protocol Dashboard](https://protocol.aragon.org)
+The Aragon Protocol Dashboard is the central app where all dispute-related information is available for guardians.
+
+#### 📚 [Read the User Guide](https://help.aragon.org/category/47-aragoncourt) 
+Read the user guide if you have any doubts about the Aragon protocol, Protocol Dashboard, or related tools.
+
+## Protocol
+
+#### 📓 [Read the full documentation](/docs)
+Aragon Protocol is a dispute resolution protocol that runs on Ethereum. It's one of the core components of the [Aragon Network](https://aragon.org/network/).
+
+#### 🚧 Project stage: v1 implementation
+After a long research and development phase, Aragon Protocol's v1 implementation has been [released](https://www.npmjs.com/package/@aragon/protocol) and [deployed](https://etherscan.io/address/0xee4650cBe7a2B23701D416f58b41D8B76b617797#code).
```

Comment:
```suggestion
After a long research and development phase, Aragon Protocol's v1 implementation will soon be released as an upgraded version of Aragon Court.
```

Reply:
Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20